### ISSUE TITLE
Busy pop should grow

### ISSUE THREAD

**Issue by Muffinman** [dark-galaxy/dark-galaxy#26]
It's not hard to manage pop growth and usage, it's just annoying, and I don't think it really adds any fun to the game.

Let's let busy pop reproduce too.

**Comment by TOI-Ingus** [comment]
I would disagree slightly in that it does create some variation in strategy, especially when looking beyond hp. Makes targeted invasion lists more workable but I guess we can see.

**Comment by Muffinman** [comment]
I agree we do need some diversification in the build orders.

The main blocker on fast-inv lists is probably the time it takes to build all the pre-requisites. We could maybe look at:

- Removing LWF as as a requirement for soldiers / inv ships to allow faster rush to invasion
- Decreasing cost / build time / pop requirement for inv ships
- Adding invasion combat score

One bonus of the pop growth change is that you now have more pop to train sols :)

**Comment by TOI-Ingus** [comment]
I think this will need to be reverted. Given the free op at the start, which should be kept, if you hit a freighter early then the pop growth in gal1 becomes too much to invade and as the pop eats more food you can't build up a good store of food either.

This wouldn't be good for invasion lists, takes you around 40 turns to go from 30-50k pop on hp with lc so physically you can have 2 freighters and 40k pop in gal1 by about turn 140-150.

**Comment by Muffinman** [comment]
Given the amount of people who have complained about this, I would like to find another way to solve this.

Also thematically it doesn't make any sense really (if that matters).

I will put the question onto Discord and see what suggestions come up.

**Comment by NewWorkPro** [comment]
One way might be to have a choice at signup to get either a inv ship or an outpost at the start (no soldiers). Then you dont need a ship yard to start invading. 

Another way to help pop growth - which is really a bit slow atm - would be to reduce the amount of pop needed to build buildings (not necessarily ships or sols) (this could also be an avenue of research in the future perhaps). 

Another idea: 
pop growth could scale, at least partially, with food output. This way, early farms would have an actual reason to build and it fits "lore wise" - surplus amenities lead to more procreation :) This would create a tradeoff to be made early between building resources to get expensive ships faster or scaling pop and food first to start building ships a bit later but then grow faster.

my 2c

- cox

**Comment by Muffinman** [comment]
Should hopefully be all working for next round!
